src/views/wait-for-others-to-sign.njk

```diff
@@ -0,0 +1,42 @@
+{% extends 'layout.njk' %}
+
+{% from 'govuk/components/breadcrumbs/macro.njk' import govukBreadcrumbs %}
+{% from 'govuk/components/button/macro.njk' import govukButton %}
+{% from "govuk/components/panel/macro.njk" import govukPanel %}
+
+{% block pageTitle %}
+  {{ serviceName }}
+{% endblock %}
+
+{% block content %}
+  <main class="govuk-main-wrapper">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+
+        <div class="govuk-panel" style="background:#1d70b8">
+          <h1 class="govuk-panel__title govuk-heading-l" style="color:#ffffff">
+             You must ask the other directors or members to sign the application
```

Comment:
Just a heads up - it was my call to use plain html here as the existing govuk component didn't allow for customization. Suggestions to make it cleaner are welcome

Reply:
Inline CSS :drake-no: 

I'm pretty sure appeals added a similar component here https://github.com/companieshouse/lfp-appeals-frontend/blob/master/src/views/other-reason-disclaimer.njk#L22

If absolutely necessary, custom CSS can be added to https://github.com/companieshouse/cdn.ch.gov.uk

Try reaching out to the Tiny Rebels to see how they went about creating their component

Reply:
Ok thanks everyone, reached out to Juliano to ask how they created a component!